ios/Podfile

```diff
@@ -0,0 +1,108 @@
+require_relative './podfile_helpers.rb'
+require_relative '../packages/react-native-unimodules/cocoapods.rb'
+
+source 'https://cdn.cocoapods.org/'
+platform :ios, '10.0'
+
+install! 'cocoapods',
+         :generate_multiple_pod_projects => true,
+         :incremental_installation => true
+
+inhibit_all_warnings!
```

Comment:
Great, thank you (and for setting `:inhibit_warnings => false` for code that we are directly responsible for).